ci-operator/step-registry/cucushift/upgrade/cpou/unpause-worker-mcp/cucushift-upgrade-cpou-unpause-worker-mcp-commands.sh

```diff
@@ -100,11 +100,5 @@ do
     printf "\n"
     unpause ${mcp}
     printf "\n"
-    if [[ "${current_minor_ver}" -lt "19" ]] && [[ $(oc get nodes -l node.openshift.io/os_id=rhel) != "" ]]; then
```

Comment:
@gpei could you help remove above line93-line94 together? It was added for checking current_minor_ver in previous pr.

Reply:
updated